src/State/Hub.php

```diff
@@ -216,6 +216,12 @@ public function getIntegration(string $className): ?IntegrationInterface
      */
     public function startTransaction(TransactionContext $context): Transaction
     {
+        $numArgs = \func_num_args();
+        $customSamplingContext = null;
+        if ($numArgs > 1) {
```

Comment:
Both add a new blank line above this one to improve readability and inline the `func_num_args()` function call